equipe_mafiarural/conceive/README.md

```diff
@@ -0,0 +1,173 @@
+<!-- PROJECT LOGO -->
+<div align="center">
+  <h3 align="center">Sistema de Monitoramento de Vazamento de Gás de Cozinha</h3>
+
+  <p align="center">
+    <strong>(Conceive)</strong>
+    <br />
+</div>
+
+## Índice
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#introdução">Introdução</a>
+    </li>
+    <li>
+      <a href="#objetivos">Objetivos</a>
+    </li>
+    <li>
+      <a href="#requisitos-principais">Requisitos Principais</a>
+    </li>
+    <li>
+      <a href="#requisitos-extras">Requisitos Extras</a>
+    </li>
+    <li>
+      <a href="#componentes-utilizados">Componentes Utilizados</a>
+      <ul>
+        <li><a href="#hardware">Hardware</a></li>
+        <li><a href="#software">Software</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#arquitetura-do-sistema">Arquitetura do Sistema</a>
+    </li>
+    <li>
+      <a href="#desenvolvimento">Desenvolvimento</a>
+      <ul>
+        <li><a href="#configuração-inicial">Configuração Inicial</a></li>
+        <li><a href="#programação">Programação</a></li>
+        <li><a href="#testes">Testes</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#pontos-chaves-do-sistema">Pontos Chaves do Sistema</a>
+      <ul>
+        <li><a href="#monitoramento">Monitoramento</a></li>
+        <li><a href="#controle">Controle</a></li>
+        <li><a href="#ações">Ações</a></li>
+        <li><a href="#visualização-e-feedback">Visualização / Feedback</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#plano-de-ação-para-implementação-de-projeto-piloto">Plano de Ação para Implementação de Projeto Piloto</a>
+    </li>
+  </ol>
+</details>
+
+
+---
+
+## Introdução
```

Comment:
Como será esse sistema?
Tentei achar no restante do texto, mas a ideia está vaga e diferente do que havia sido conversado.
Está faltando algum diagrama para apresentar a ideia da organização do produto.
Como os sensores serão espalhados na casa? E Por quê?
Qual(is) válvula(s) serão cortadas? Haviamos conversado para isso ser melhor especificado, mas não foi colocado aqui.
Será usado apenas um ESP32? Vai ser tudo conectado por fio? Isso não é IoT.
Além disso, não vi nada sobre IoT no projeto.